requirements-jax.txt

```diff
@@ -1,2 +1,3 @@
+chex
```

Comment:
I think that for now (in this PR) we should limit `chex` to testing, and create our own types for `Array` and `PRNGKey`. The last two are too simple to justify a dependency of the _library_ (not the tests) on `chex`. In which case the dependency name would go in `requirements-dev.txt`.

Reply:
Sure, `Array` and `PRNGKey` are both pretty simple type alias. 
We can make a decision later whether we want chex as main dependency (e.g., is the Dataclass really that useful)

Reply:
Sure, that sounds good.

Reply:
Let's deal with the types in #79 then.

Reply:
+1, I will wait for #79 to merge